server/routes/profileRouter.js

```diff
@@ -1,13 +1,18 @@
 const express = require("express");
 const router = express.Router();
 const profileController = require("../controllers/profileController");
+const { validateProfile, validateSitterProfile } = require("../validate");
 // uploadController doesn't exist in the current main branch.
 // const uploadController = require("../controllers/uploadController");
-
-router.post("/", profileController.createProfile);
-router.put("/update", profileController.updateProfile);
-router.get("/", profileController.findProfile);
-router.get("/all", profileController.getAllProfiles);
+router.post("/", validateProfile, profileController.createProfile);
```

Comment:
I think we can add the protect middleware to the applicable routes here? :)

Reply:
@rajivtitus The `protect` middleware is added to the applicable routes in `app.js`. I tested them with Postman. (I guess it worked?) Where is the most suitable place to add `protect` middleware between `app.js` and `somethingRoute.js`? 🤔

Reply:
@jstyle5 I may not have noticed that, having it in the app.js is completely fine. However in some cases a few of our routes may not need to be protected (to take these cases into consideration it would be better to add the middleware to each individual route within the router)

Reply:
> @jstyle5 I may not have noticed that, having it in the app.js is completely fine. However in some cases a few of our routes may not need to be protected (to take these cases into consideration it would be better to add the middleware to each individual route within the router)

@rajivtitus it would be better to add the middleware to each individual route within the router <-- Okay, thank you. I will move `protect` to each individual route when I merge. 🙂

Reply:
Applied